flash-tools.nix

```diff
@@ -26,7 +26,7 @@ let
       perl
     ];
 
-    patches = [ ./flash-tools.patch ./flash-tools-secureboot.patch ];
+    patches = [ ./flash-tools.patch ./flash-tools-secureboot.patch ./gen_uefi_default_keys.patch ];
```

Comment:
This file is being so extensively patched, I'd rather we not use their script at all and just create the dts file and compile it ourselves. The structure of the device tree overlay isn't so complicated we can't just do it ourselves.